commit 5723023bc88508bae3fd99b706d9ecfd1e429c26
Author: Sjoerd Simons <sjoerd@luon.net>
Date:   Wed Oct 18 23:06:28 2006 +0200

    fix bashisms in the various hal callout scripts
    
    Some of the hal callout scripts use == for string comparison.
    Which is a bashism and will make the scripts fail if /bin/sh isn't
    bash.

diff --git a/tools/hal-system-power-hibernate b/tools/hal-system-power-hibernate
index ab8ce9d..ffdd330 100755
--- a/tools/hal-system-power-hibernate
+++ b/tools/hal-system-power-hibernate
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRIVILEGE=hal-power-hibernate
-if [ "$HAVE_POLKIT" == "1" ] ; then
+if [ "$HAVE_POLKIT" = "1" ] ; then
     if [ "$HAL_METHOD_INVOKED_BY_UID" != "0" ] ; then
 	RESULT=$(polkit-is-privileged --privilege $PRIVILEGE \
 	    --user $HAL_METHOD_INVOKED_BY_UID \
diff --git a/tools/hal-system-power-reboot b/tools/hal-system-power-reboot
index dec4b0b..1c93aac 100755
--- a/tools/hal-system-power-reboot
+++ b/tools/hal-system-power-reboot
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRIVILEGE=hal-power-reboot
-if [ "$HAVE_POLKIT" == "1" ] ; then
+if [ "$HAVE_POLKIT" = "1" ] ; then
     if [ "$HAL_METHOD_INVOKED_BY_UID" != "0" ] ; then
 	RESULT=$(polkit-is-privileged --privilege $PRIVILEGE \
 	    --user $HAL_METHOD_INVOKED_BY_UID \
diff --git a/tools/hal-system-power-shutdown b/tools/hal-system-power-shutdown
index 33a3226..df3d14a 100755
--- a/tools/hal-system-power-shutdown
+++ b/tools/hal-system-power-shutdown
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRIVILEGE=hal-power-poweroff
-if [ "$HAVE_POLKIT" == "1" ] ; then
+if [ "$HAVE_POLKIT" = "1" ] ; then
     if [ "$HAL_METHOD_INVOKED_BY_UID" != "0" ] ; then
 	RESULT=$(polkit-is-privileged --privilege $PRIVILEGE \
 	    --user $HAL_METHOD_INVOKED_BY_UID \
diff --git a/tools/hal-system-power-suspend b/tools/hal-system-power-suspend
index 8199689..bd698b7 100755
--- a/tools/hal-system-power-suspend
+++ b/tools/hal-system-power-suspend
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRIVILEGE=hal-power-suspend
-if [ "$HAVE_POLKIT" == "1" ] ; then
+if [ "$HAVE_POLKIT" = "1" ] ; then
     if [ "$HAL_METHOD_INVOKED_BY_UID" != "0" ] ; then
 	RESULT=$(polkit-is-privileged --privilege $PRIVILEGE \
 	    --user $HAL_METHOD_INVOKED_BY_UID \
diff --git a/tools/linux/hal-system-power-suspend-linux b/tools/linux/hal-system-power-suspend-linux
index 59cde73..6842cc0 100755
--- a/tools/linux/hal-system-power-suspend-linux
+++ b/tools/linux/hal-system-power-suspend-linux
@@ -19,7 +19,7 @@ unsupported() {
 read seconds_to_sleep
 
 #PMU systems cannot use /sys/power/state yet, so use a helper to issue an ioctl
-if [ "$HAL_PROP_LAPTOP_PANEL_ACCESS_METHOD" == "pmu" ]; then
+if [ "$HAL_PROP_LAPTOP_PANEL_ACCESS_METHOD" = "pmu" ]; then
 	hal-system-power-pmu sleep
 	if [ $? -ne 0 ]; then
 		echo "org.freedesktop.Hal.Device.SystemPowerManagement.NotSupported" >&2
